ebusd-2.1.x/de/vaillant/76.vwzio.HW5103.SW0902.csv

```diff
@@ -0,0 +1,76 @@
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment,field2,part (m/s),datatypes/templates,divider/values,unit,comment,field3,part (m/s),datatypes/templates,divider/values,unit,comment,field4,part (m/s),datatypes/templates,divider/values,unit,comment
+
+# START Unbekannte Meldungen alle 10 Sekunden,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+r,vwz,unknownMPoll10sB512h,,,76,B512,0f0001,,,HEX:7,,,Unbekannte Abfrage alle 10 Sekunden von CTLV2 an VWZ,,,,,,,,,,,,,,,,,,
+# END Unbekannte Meldungen alle 10 Sekunden,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+
+*r,,,,,,B514,05,,,IGN:2,,,,,,,,,,,,,,,,,,,,,
+# alle Nachrichten entfernt die ffff ergeben haben,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+
+r,,BuildingCircuitPumpPowerSensor,,,,,0103FFFF,,,UIN,,,Pumpenleistung des Gebäudekreislaufs (Prozent des Maximum),,,,,,,,,,,,,,,,,,
+r,,ThreeWayValveStatus,,,,,0203FFFF,,,UCH,0=heating;1=dhw,,Dreiwegeventil (Heizung/Warmwasser),,,,,,,,,,,,,,,,,,
+r,,xUnknown11,,,,,1103FFFF,,,HEX:*,,,,,,,,,,,,,,,,,,,,,
```

Comment:
if it is unknown, then comment it out at least as to not get unwanted messages in the first place. same for the other translation I guess